pom.xml

```diff
@@ -1445,6 +1445,12 @@
                 <groupId>com.esotericsoftware.kryo</groupId>
                 <artifactId>kryo</artifactId>
                 <version>2.21</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.ow2.asm</groupId>
+                        <artifactId>asm</artifactId>
```

Comment:
nice, can we add a comment to the exclusion block so we remember that in the future?